app/models/expense.rb

```diff
@@ -1,6 +1,13 @@
 class Expense < ActiveRecord::Base
   belongs_to :budget
 
-  validates :name, presence: true, uniqueness: true
-  validates :frequency, presence: true
+  enum frequency: [:weekly, :fortnightly, :monthly, :quarterly, :annually]
```

Comment:
I'm usually a fan of adding a numerical key to enums, so that there will not be a problem in the future to add new keys, without issues to key order and existing records data.

In this case, new frequencies can only be added in the end of the array, or existing values will clash.

Reply:
Good point! Done in sha: 0132e1f